presto-product-tests/bin/run_on_docker.sh

```diff
@@ -83,6 +83,8 @@ function stop_application_runner_containers() {
     docker stop ${CONTAINER_NAME}
     echo "Container stopped: ${CONTAINER_NAME}"
   done
+  echo "Removing dead application-runner containers"
+  docker ps -aq --no-trunc --filter status=dead --filter status=exited | xargs docker rm -v || true
```

Comment:
I mean other stopped / exited containers (and their volumes) not started by run_on_docker.sh. Like, I run the product tests and BAM my personal blog disappears ;)